test/AngularSpec.js

```diff
@@ -7,13 +7,38 @@ Float32Array, Float64Array,  */
 
 describe('angular', function() {
   var element, document;
+  var originalObjectMaxDepthInErrorMessage = minErrConfig.objectMaxDepth;
 
   beforeEach(function() {
     document = window.document;
   });
 
   afterEach(function() {
     dealoc(element);
+    minErrConfig.objectMaxDepth = originalObjectMaxDepthInErrorMessage;
+  });
+
+  describe('errorHandlingConfig', function() {
+    it('should get default objectMaxDepth', function() {
+      expect(errorHandlingConfig().objectMaxDepth).toBe(5);
+    });
+
+    it('should set objectMaxDepthx', function() {
+      errorHandlingConfig({objectMaxDepth: 3});
+      expect(errorHandlingConfig()).toEqual({objectMaxDepth: 3});
```

Comment:
To make this test more robust, use `expect(errorHandlingConfig().objectMaxDepth).toBe(3)`.
So, if we introduce more properties in the future, it won't break.